src/Control/Algebra.hs

```diff
@@ -103,8 +105,8 @@ handleCoercible = handleIdentity coerce
 instance Algebra (Lift IO) IO where
   alg = join . unLift
 
-instance Algebra Pure Identity where
-  alg v = case v of {}
+instance Algebra (Lift Identity) Identity where
+  alg = join . unLift
```

Comment:
When I wrote the `Algebra` instance for `IO`, above, @patrickt asked if it wouldn’t hold for any `Monad` m that you could write an `Algebra (Lift m) m` instance—and indeed it would!